api/src/backend/api/models.py

```diff
@@ -1370,6 +1372,26 @@ class Meta:
         ]
 
 
+class SAMLToken(models.Model):
```

Comment:
No RLS at all?

Reply:
No, we’re not logged in at this point, so we need it to work without RLS